src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/organization_policy.py

```diff
@@ -35,12 +35,9 @@ def _compare_ordered_policy(self, obj):
             return obj
 
     @staticmethod
-    def _trim_scp_file_name(policy):
-        return policy[1:][:-8] if policy[1:][:-8] == '/' else policy[2:][:-9]
-
-    @staticmethod
-    def _trim_tagging_policy_file_name(policy):
-        return policy[1:][:-19] if policy[1:][:-19] == '/' else policy[2:][:-20]
+    def _trim_policy_file_name(policy):
+        # Returns policy target and filename as tuple
+        return '/' if '/'.join(policy.split('/')[1:-1]) == '' else '/'.join(policy.split('/')[1:-1]), policy.split('/')[-1]
```

Comment:
This code looks to complex for what it is doing I am afraid.

Could you change this function to use a normal if, instead of an inline one.
And could you use variables and add comments to explain what is happening exactly?

It would be good to add tests for this too, inside the `src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/tests/` directory. Let me know if you need help with that, as this file is not tested at all I see.
I'm trying to raise our test coverage, as debugging/testing all use cases of ADF is extremely hard to accomplish otherwise.

Reply:
I simplified/commented this routine.

As far as tests go, I may need some help getting that started for this file. If you could do get a basic test file for organization_policy.py started, I can help expand upon that.

Reply:
I've added in a basic test that tests the policy name function.